packages/stylex/src/StyleXTypes.js

```diff
@@ -176,12 +177,13 @@ export type Theme<
   [string]: StyleXClassNameFor<string, IDFromVarGroup<T>>,
 }>;
 
-export type OverridesForTokenType<Config: { +[string]: mixed }> = {
+export type OverridesForTokenType<Config extends { [key: string]: unknown }> = {
   [Key in keyof Config]:
     | Config[Key]
-    | { +default: Config[Key], +[string]: Config[Key] },
+    | { default: Config[Key]; [property: string]: Config[Key] };
```

Comment:
Revert this line.
`+` means `readonly` in Flow and is needed.
`[string]` is valid in Flow, and there is no need for the `property:` bit.
Please bring back the `+` here.

Reply:
Okay I will correct this

Reply:
What is the purpose of your PR? Every change this PR breaks Flow by trying to change it to Typescript syntax.

Reply:
@nmn  I thought the problem occurred , can be fixed by changing the type, it was my first try to contribute to open source I think I am not at the ryt place then.
Thanks

Reply:
I will try to fix what's bad in my PR, I am sure that I'll learn something :)

Reply:
@develop084 From what I understand, you're trying to change the types of `createTheme` to allow changing *some* of the variables instead of all of them.

I've already made this fix and it'll be out in the next release in a few days.

----

I don't want to discourage you from learning and contributing. I would suggest reading the Contribution.md document, and learn more about Flow at flow.org

Then, make sure you have the Flow plugin for VSCode installed and functional before you make your next attempt.